tracking-migration/src/test/java/io/customer/tracking/migration/testutils/data/MigrationDataSet.kt

```diff
@@ -224,3 +224,68 @@ sealed interface DeletePushNotificationQueueTaskData : MigrationDataSet {
         """.trimIndent()
     }
 }
+
+sealed interface InvalidMigrationDataSet : MigrationDataSet {
```

Comment:
```
        fun createJsonString(storageId: String, type: String, data: String, totalRuns: Int): String = """
            {
              "storageId": "$storageId",
              "type": "$type",
              "data": $data,
              "runResults": {
                "totalRuns": $totalRuns
              }
            }
        """.trimIndent()
    }
```

its a one off file so doesn't really matter, but something like this could have made life a little easier 😅

Reply:
Yeah, but if we have done this in initial PR 😄 Since there are a lot of similar json samples in this file, I would stick with current implementation for now as all json samples will need to be updated to use this method. We can improve this later to look more clean.